doc/cabal-package-description-file.rst

```diff
@@ -2671,37 +2671,22 @@ Source Repositories
 .. pkg-section:: source-repository
     :since: 1.6
 
-It is often useful to be able to specify a source revision control
-repository for a package. Cabal lets you specify this information in
-a relatively structured form which enables other tools to interpret and
-make effective use of the information. For example the information
-should be sufficient for an automatic tool to checkout the sources.
+Where to find the source for this package.
 
-Cabal supports specifying different information for various common
-source control systems. Obviously not all automated tools will support
-all source control systems.
-
-Cabal supports specifying repositories for different use cases. By
-declaring which case we mean automated tools can be more useful. There
-are currently two kinds defined:
+There are two kinds and you can specify one or the other or both:
```

Comment:
```suggestion
There are two kinds, and you can specify one or the other or both:
```

Reply:
Grammarly disliked the second clause so I made it clearer and then ended up splitting the sentence in two. Thanks for the suggestion @geekosaur.